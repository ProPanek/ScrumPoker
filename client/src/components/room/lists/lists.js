import React, {Component} from "react"
import {inject, observer} from "mobx-react";
import {withRouter} from "react-router-dom";
import {decorate, observable} from "mobx";
import UserList from "./user_list/user_list";
import HistoryList from "./history_list/history_list";

class Lists extends Component {
  render() {
    return (
      <React.Fragment>
        <UserList/>
        <HistoryList/>
      </React.Fragment>
    )
  }
}

decorate(Lists, {
  userId: observable,
});

export {Lists}
export default inject("store")(withRouter(observer(Lists)));

