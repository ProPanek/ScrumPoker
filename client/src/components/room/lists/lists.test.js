import React from "react"
import {Lists} from "./lists"
import { shallow } from "enzyme";

describe("UserList", () => {
  it("renders without errors", () => {

    const component = shallow(<Lists />)
    expect(component).toMatchSnapshot()
  })
})
