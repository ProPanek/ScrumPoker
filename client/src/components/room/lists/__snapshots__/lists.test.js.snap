// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserList renders without errors 1`] = `
<Fragment>
  <inject-withRouter(UserList)-with-store />
  <inject-HistoryList-with-store />
</Fragment>
`;
